### PR TITLE
Animated games page

### PR DESCRIPTION
**Pull request by L5050**

### CODE REVIEW

File: src/routes/games/+page.svelte [hunk 1]
```diff
@@ -1,79 +1,81 @@
...
+                </a>
+            </li>
+        {/each}
+    </ol>
+</main>
 
 <style>
+    .w-full {
+        padding: 30px;
+    }
```
Comment (bates64) [inline src/routes/games/+page.svelte]:
```suggestion
```

File: src/routes/games/+page.svelte [hunk 2]
```diff
@@ -1,79 +1,81 @@
...
-    </main>
-
-    <p class="text-sm text-gray-400">
-        Box art courtesy of <a
-            href="https://gamesdb.launchbox-app.com/"
-            class="hover:underline">LaunchBox Games Database</a
-        >
-    </p>
-</div>
+                    <div class="w-full">
```
Comment (bates64) [inline src/routes/games/+page.svelte]:
```suggestion
                    <div class="w-full p-4">
```

File: src/routes/games/+page.svelte [hunk 3]
```diff
@@ -1,79 +1,81 @@
...
+        background-color: white;
+        border-radius: 15px;
+        box-shadow: 0 2px 5px rgba(0, 0, 0, 0.15);
+        text-decoration: none;
+    }
 
-        text-wrap: balance;
+    .game img:hover {
+        transform: rotate3d(0, 1, 0, 35deg);
+    }
```
Comment (bates64) [inline src/routes/games/+page.svelte]: Maybe better as `.game:hover img`

File: src/routes/games/+page.svelte [hunk 4]
```diff
@@ -1,79 +1,81 @@
...
 
-        &:hover,
-        &:focus {
-            &[href] img {
-                /*transform: var(--base-transform) rotate3d(0, 1, 0, 5deg) translate3d(50%, 0, 60%);*/
-            }
-        }
+    .game p {
+        margin-top: 4px;
+        color: #718096;
```
Comment (bates64) [inline src/routes/games/+page.svelte]: Use `text-gray-700` or something

File: src/routes/games/+page.svelte [hunk 5]
```diff
@@ -1,79 +1,81 @@
-<script lang="ts">
+<script>
```
Comment (bates64) [inline src/routes/games/+page.svelte]: Why remove ts?

File: src/routes/games/+page.svelte [hunk 6]
```diff
@@ -1,79 +1,81 @@
...
-        position: relative;
+        display: flex;
+        flex-direction: column;
+        justify-content: start;
+        align-items: start;
+        padding: 4px;
+        background-color: white;
+        border-radius: 15px;
+        box-shadow: 0 2px 5px rgba(0, 0, 0, 0.15);
+        text-decoration: none;
```
Comment (bates64) [inline src/routes/games/+page.svelte]: These can and should all be tailwind

`flex flex-col justify-start items-start p-2 bg-white rounded-md shadow no-underline`

**Review by bates64 — CHANGES_REQUESTED** [review]

File: src/routes/games/+page.svelte (hunk 5 above)
Comment (L5050) [reply src/routes/games/+page.svelte]: accident

**Review by L5050 — COMMENTED** [review]
completed requested changes

**Review by bates64 — APPROVED** [review]

**Pull request merged**